shell/common/shell.cc

```diff
@@ -899,6 +899,12 @@ void Shell::OnPlatformViewDestroyed() {
   // Overall, the longer term plan is to remove this implementation once
   // https://github.com/flutter/flutter/issues/96679 is fixed.
   rasterizer_->TeardownExternalViewEmbedder();
+
+  task_runners_.GetUITaskRunner()->PostTask([engine = engine_->GetWeakPtr()]() {
```

Comment:
Would it be safe to move this to the start of `Shell::OnPlatformViewDestroyed` so it can run concurrently with the IO and raster thread tasks in that method?

Reply:
Moved to the start.